modules/ossm-installing-operator.adoc

```diff
@@ -26,8 +26,20 @@ For clusters without {SMProduct} instances, install the {SMProductShortName} Ope
 
 . When the prompt that discusses the community operator appears, click *Continue*.
 
-. Verify the {SMProductShortName} Operator is version 3.0, and click *Install*.
+. Click *Install*.
 
-. Use the default installation settings presented, and click *Install* to continue.
+. On the *Install Operator* page, perform the following steps:
+
+.. Select *All namespaces on the cluster (default)* as the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be available to all namespaces in the cluster.
+
+.. Select *Automatic* as the *Approval Strategy*. This ensures that the {olm-first} handles the future upgrades to the Operator automatically. If you select the *Manual* approval strategy, {olm} creates an update request. As a cluster administrator, you must then manually approve the {olm} update request to update the Operator to the new version.
+
+.. Select an *Update Channel*.
++
+* Choose the *stable* channel to install the latest stable version of the {SMProductName} 3 Operator. It is the default channel for installing the Operator.
++
+* TTo install a specific version of the {SMProductName} 3 Operator, choose the corresponding `stable-<version>` channel. For example, to install the {SMProductName} Operator version 3.0.x, use the *stable-3.0* channel.
+
+. Click *Install* to install the Operator.
 
 . Click *Operators* -> *Installed Operators* to verify that the {SMProductShortName} Operator is installed. `Succeeded` should appear in the *Status* column.
```

Comment:
.Verification

Click *Operators* -> *Installed Operators* to verify that the {SMProductShortName} Operator is installed. `Succeeded` should show in the *Status* column.